src/Resources/public/dist/entrypoints.json

```diff
@@ -0,0 +1,15 @@
+{
+  "entrypoints": {
+    "sonata_admin": {
+      "js": [
+        "/bundles/sonataadmin/dist/runtime.js",
+        "/bundles/sonataadmin/dist/2.js",
+        "/bundles/sonataadmin/dist/sonata_admin.js"
+      ],
+      "css": [
+        "/bundles/sonataadmin/dist/2.css",
+        "/bundles/sonataadmin/dist/sonata_admin.css"
+      ]
+    }
+  }
+}
```

Comment:
Please add new line

Reply:
this is generated

Reply:
Argh this should not happen, do you know where we need to head to provide a PR?

Reply:
cc @weaverryan

Reply:
friendly ping @weaverryan

Reply:
@OskarStark i figured out: 

`manifest.json` is generated: https://github.com/symfony/webpack-encore/blob/master/lib/plugins/entry-files-manifest.js#L80

But for `entrypoints.json` i didn't find the place where that happens